src/ui/actions/timeline.ts

```diff
@@ -264,6 +269,44 @@ export function seek(
   };
 }
 
+export function jumpToReduxCode(
```

Comment:
I'm not sure if this makes more sense as a redux action or as a suspense cache that returns the frameStep that we can call `seek` on

Reply:
Yeah, the main logic here could be moved into its own cache.